numpy-stubs/__init__.pyi

```diff
@@ -773,10 +773,10 @@ class matrix:
 #
 # Specific values
 #
-inf: float
-nan: float
-NaN: float
-NAN: float
+inf: float32
+nan: float32
+NaN: float32
+NAN: float32
```

Comment:
this doesn't seem true?

```python
type(np.inf) == float
```

Reply:
yeah....I was trying to get around a problem by introducing a new problem/error. Resolved